.github/workflows/macos_install.sh

```diff
@@ -70,7 +70,7 @@ export CPPFLAGS="-isystem${CONDA_PREFIX}/include"
 ./configure $CONFIGURE_FLAGS
```

Comment:
```suggestion
./configure "$CONFIGURE_FLAGS"
```

Reply:
This I wasn't sure that it would work on posix shell on Mac, and didn't know the complete side effects

Reply:
What spellcheck suggest is the same on Mac.